gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -108,6 +119,7 @@ def __init__(
     self._allow_incompatible_records = allow_incompatible_records
     self._omit_empty_sample_calls = omit_empty_sample_calls
     self._num_bigquery_write_shards = num_bigquery_write_shards
+    self._update_bigquery_schema(update_schema)
```

Comment:
Please check if update_schema is true before calling this method and remove the check from inside the function (I think you can also assume that self._schema exists; otherwise, the method above would have returned an error).